crates/starknet-server/tests/get_transaction_receipt_by_hash.rs

```diff
@@ -115,15 +116,20 @@ mod get_transaction_receipt_by_hash_integration_tests {
             .await
             .unwrap();
 
-        let result: starknet_rs_core::types::MaybePendingTransactionReceipt = devnet
+        let result: MaybePendingTransactionReceipt = devnet
             .json_rpc_client
             .get_transaction_receipt(declare_transaction.transaction_hash)
             .await
             .unwrap();
 
         match result {
-            MaybePendingTransactionReceipt::Receipt(TransactionReceipt::Declare(declare)) => {
-                assert_eq!(declare.status, TransactionStatus::Rejected);
+            MaybePendingTransactionReceipt::PendingReceipt(PendingTransactionReceipt::Declare(
+                declare,
+            )) => {
+                assert_eq!(
+                    declare.execution_result.revert_reason(),
+                    Some("Invalid transaction nonce. Expected: 0 got 1")
```

Comment:
```suggestion
                    Some("Invalid transaction nonce. Expected: 0, got 1")
```

Reply:
This msg depends on starknet_in_rust. Following changes make tests crash

Reply:
Reverted